docs/ref/settings.txt

```diff
@@ -854,6 +854,23 @@ TEST_TBLSPACE_TMP
     Use the :setting:`TBLSPACE_TMP <TEST_TBLSPACE_TMP>` entry in the
     :setting:`TEST <DATABASE-TEST>` dictionary.
 
+.. setting:: DATA_UPLOAD_MAX_MEMORY_SIZE
+
+DATA_UPLOAD_MAX_MEMORY_SIZE
+---------------------------
+
+.. versionadded:: 1.8
+
+Default: ``2621440`` (i.e. 2.5 MB).
+
+The maximum size (in bytes) that a request body may be before raising a
+:exc:`~django.core.exceptions.SuspiciousOperation` error. This value is applied
+when accessing ``request.body`` or ``request.POST`` and is calculated against
+the total request size excluding any file upload data. You can set it to
+``None`` to disable the check.
```

Comment:
Would it be worth also adding a note recommending that users consider configuring a maximum upload size on their server as well, where possible?  (Not always an option if eg on a managed hosting platform, but more suitable in other cases, with the application level restriction being our general purpose fall-back)